src/elements/content-sidebar/activity-feed/task-new/TaskActions.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { ACTIVITY_TARGETS } from '../../../common/interactionTargets';
+import messages from '../../../common/messages';
+import Button from '../../../../components/button';
+import { TASK_TYPE_APPROVAL } from '../../../../constants';
+
+type Props = {
+    onTaskApproval: Function,
+    onTaskComplete: Function,
+    onTaskReject: Function,
+    taskType?: TaskType,
+};
+
+const TaskActions = ({ onTaskApproval, onTaskReject, onTaskComplete, taskType }: Props): React.Node => (
```

Comment:
This looks like it may be a duplicate of the other TaskActions component.

Reply:
what other taskactions?